test/Driver/batch_mode_remark.swift

```diff
@@ -0,0 +1,12 @@
+// Ensure that driver issues a remark iff in batch mode.
+//
+// RUN: %swiftc_driver -whole-module-optimization -enable-batch-mode   %S/../Inputs/empty.swift -### 2>&1 | %FileCheck %s
+// RUN: %swiftc_driver                                                 %S/../Inputs/empty.swift -### 2>&1 | %FileCheck %s
+// RUN: %swiftc_driver -enable-batch-mode        -disable-batch-mode   %S/../Inputs/empty.swift -### 2>&1 | %FileCheck -allow-empty %s
```

Comment:
I'm a little surprised only one of these is empty. Are the others emitting some *other* diagnostic?